alembic/versions/2022-08-05-17.07.07_089bfa50d0ed_add_is_ocr_recipe_column_to_recipes.py

```diff
@@ -1,7 +1,7 @@
 """Add is_ocr_recipe column to recipes
 
 Revision ID: 089bfa50d0ed
-Revises: f30cf048c228
+Revises: 188374910655
```

Comment:
FYI this is just a change to the comment, not the actual migration looks like I forgot to edit it in #1619